saleor/graphql/api.py

```diff
@@ -286,24 +287,24 @@ class Mutations(graphene.ObjectType):
     category_delete = CategoryDelete.Field()
     category_update = CategoryUpdate.Field()
 
+    collection_create = CollectionCreate.Field()
+    collection_update = CollectionUpdate.Field()
+    collection_delete = CollectionDelete.Field()
+    collection_products_add = CollectionProductsAdd.Field()
```

Comment:
As suggested [in this comment](https://github.com/mirumee/saleor/issues/2757#issuecomment-426213619) I would consider "collection" to be the entity. So the original name was correct.